mpt/impl/managesieve/core/src/main/java/org/apache/james/mpt/host/JamesManageSieveHostSystem.java

```diff
@@ -35,9 +35,9 @@
     protected final SieveRepository sieveRepository;
     private final ManageSieveProcessor processor;
 
-    public JamesManageSieveHostSystem(UsersRepository usersRepository, SieveRepository sieveRepository) throws Exception {
-        this.usersRepository = usersRepository;
-        this.sieveRepository = sieveRepository;
+    public JamesManageSieveHostSystem() throws Exception {
+        this.usersRepository = createUsersRepository();
```

Comment:
calling abstract methods into a constructor is an anti-pattern : the method you call can't know which fields are initialized.